### PR TITLE
Feature/jsunittest

### PR DESCRIPTION
**Pull request by tgjonestx**
See web/README.md for usage details.

FYI - I created new tests to show that it all worked. I tried to pull over low hanging fruit from the previous tests. However, I left the ones that were more involved just for future reference - see web/static/test/obsoleteSpec.js

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -55,6 +55,12 @@ isvcs/build/*.tar.gz
 web/node_modules/
 web/static/js/controlplane.js
 web/static/js/controlplane.min.js
+<<<<<<< HEAD
+web/static/thirdparty/thirdparty.js
+web/static/thirdparty/thirdparty.min.js
+web/static/test/results
+=======
 web/static/js/controlplane.js.map
```
Comment (rDr4g0n) [inline .gitignore]: whoops... bad merge here.

File: web/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+         |
+         +-- thirdparty  thirdparty libraries used by the UI
+         |
+         +-- test        unit-test code for the UI javascript
+ ```
+
+## Build Setup
+Nothing special is required to build the code in `serviced/web`. The toplevel
+makefile for serviced calls the `serviced/web/makefile`  By default, the makefile uses
+the Docker container `zenoss/serviced-build` defined in `serviced/build/Dockefile` to launch
```
Comment (rDr4g0n) [inline web/README.md]: type in the word "Dockerfile"
    defined in `serviced/build/Dockefile`

File: web/static/js/modules/miscUtils.js [hunk 3]
```diff
@@ -225,6 +225,7 @@
 
             // TODO - use angular $location object
             unauthorized: function() {
+                // FIXME: Need to injecct window.location to make this testable
```
Comment (rDr4g0n) [inline web/static/js/modules/miscUtils.js]: Funny you should add this FIXME... look at the comment just above...

Comment (tgjonestx) [inline web/static/js/modules/miscUtils.js]: I consolidated the 2 comments

File: web/README.md (hunk 2 above)
Comment (tgjonestx) [inline web/README.md]: fixed

File: .gitignore (hunk 1 above)
Comment (tgjonestx) [inline .gitignore]: DOH!

**Pull request merged**